docker/Dockerfile.pdo-dev

```diff
@@ -89,7 +88,6 @@ RUN apt-get update \
     gnupg \
     libcurl4-openssl-dev \
     liblmdb-dev \
-    libprotobuf-dev \
```

Comment:
same here... for example, do we still need libsecp256k1-dev?

Reply:
Yes, because we still have the secp256k1 python package, which is used in the TransactionKeys.

For the record, I think that once we drop the python package we can also remove the dev library. In other circumstances we use openssl for that.

Reply:
Again... future issue... should we just replace the python transaction keys (which i think is a useful class) with an openssl implementation? Then get rid of the secp256k1-dev package completely.